src/include/denc.h

```diff
@@ -170,9 +170,9 @@ struct denc_traits {
 #define WRITE_RAW_DENC(type)						\
   template<>								\
   struct denc_traits<type> {						\
-    enum { supported = 2 };						\
-    enum { featured = false };						\
```

Comment:
@liewegas do we have a test to reproduce the clash? if yes, so does "make check" proves that we are immune from it now? if not, could you enlighten me how to create a reproducer? thanks.